README.md

```diff
@@ -106,6 +106,52 @@ buildctl build ... --exporter=local --exporter-opt output=path/to/output-dir
 buildctl du -v
 ```
 
+#### Running containerized buildkit
+
+buildkit daemon supports communicating with clients over TCP. This is supports some use cases like:
+
+* Docker for Mac or Windows users which cannot run natively buildkit daemon on their platform,
```

Comment:
add "yet", as we definitely want to support running `buildd` on windows in the future.